pyquil/api/quantum_computer.py

```diff
@@ -185,3 +189,165 @@ def wait_for_job(self, job_id):
 
     def __str__(self):
         return self.name
+
+
+def list_quantum_computers(connection: ForestConnection = None, qpus=True, qvms=True) -> List[str]:
```

Comment:
spaces in `ForestConnection = None`

Reply:
pycharm puts spaces there (?)

Reply:
PEP8:
When combining an argument annotation with a default value, use spaces around the = sign (but only for those arguments that have both an annotation and a default).

Reply:
Oh my bad I missed the type hint